examples/webrx/js/app.js

```diff
@@ -0,0 +1,156 @@
+/*global wx */
+(function () {
+	'use strict';
+    
+    var localStorageKey = 'todos-webrx';
+    var displayModeAll = "all";
+    var displayModeActive = "active";
+    var displayModeCompleted = "completed";
```

Comment:
Could you run the style checkers on this? There are a few things like double vs single quotes here.